SRE/Network-CFTS/vpc-squid.yml

```
@@ -313,8 +313,16 @@ Resources:
             yum update -y --security
 
             # Disable source / destination check. It cannot be disabled from the launch configuration
-            region=${AWS::Region}
-            instanceid=`curl -s http://169.254.169.254/latest/meta-data/instance-id`
+            # Get the session token
+            TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)